applications/DeepSpeed-Chat/README.md

```diff
@@ -239,6 +239,13 @@ If you have downloaded huggingface datasets manually, you can add your local pat
 
 One thing to note that some datasets may only have one response instead of two responses. For those datasets, you can only use them in step 1. And in such case, you should add the dataset_name as part of the "--sft_only_data_path" arg instead of the "--data_path" arg. One thing to note is that: If you plan to only do step 1 SFT, adding more single-response datasets is definitely beneficial. However, if you do plan to do steps 2 and 3, then adding too many single-response datasets during SFT could backfire: these data could be different from the data used for steps 2/3, generating different distributions which could cause training instability/worse model quality during step 2/3. That is part of the reason why we focused on trying the datasets with two responses and the preference, and always split a dataset into all 3 steps.
 
+If you have your own dataset in local files, you can also use it by following these rules:
+* Use "local/jsonfile" as the dataset name.
+* Put your train data and evaluation data in applications/DeepSpeed-Chat/data/ with name train.json and eval.json.
+* The json data in file should be a single list with each item like ***{"prompt":"\n\nHuman:aaa.\n\nAssistant:\n\n","chosen":"bbb","rejected":"ccc"}***.
```

Comment:
"\n\nHuman:aaa.\n\nAssistant:\n\n" is not a fixed format that all users need to follow. In fact if you look at other datasets we ported, they don't have "\n\n". I recommend changing it to "Human: aaa Assistant:"